cluster/gce/gci/configure-helper.sh

```diff
@@ -48,16 +48,32 @@ function create-dirs {
   fi
 }
 
+# Formats the given device ($1) if needed and mounts it at given mount point
+# ($2).
+function safe-format-and-mount() {
+  device=$1
+  mountpoint=$2
+
+  # Format only if the disk is not already formatted.
+  if ! tune2fs -l "${device}" ; then
```

Comment:
Also, this will reformat things that are XFS formatted, which the original script did not do...